app/assets/stylesheets/home.scss

```diff
@@ -3,7 +3,9 @@ $black: #000;
 $white: #ffffff;
 
 .homepage {
-  background: url("light-person-woman-fire.jpeg") no-repeat center center fixed;
+  background-image: image-url("light-person-woman-fire.jpeg");
+  background-repeat: no-repeat;
```

Comment:
Properties should be ordered background-image, background-position, background-repeat, background-size